src/pages/events/dialogs/new.tsx

```diff
@@ -136,6 +142,11 @@ export const EventNewIncidentDialog: React.FC<EventNewIncidentDialogProps> = ({
   const onChangeIncidentTeam = useCallback(
     (e: React.ChangeEvent<HTMLSelectElement>) => {
       const newTeam = teams?.find((t) => t.number === e.target.value);
+      if (e.target.value === "-1") {
+        setMatch(null);
+        setTeam(null);
+      }
```

Comment:
I am not sure about what the expected behaviour should be here.

We want to handle the user selecting 'Pick a team' in some way. At the moment it will reset both match and team to null and display 'pick a team' and 'pick a match.